common/src/main/java/com/wynntils/core/managers/Models.java

```diff
@@ -98,6 +99,7 @@ public class Models {
     public static final SkillIconPropertyModel SkillIconProperty = new SkillIconPropertyModel();
     public static final SkillPointPropertyModel SkillPointProperty = new SkillPointPropertyModel();
     public static final SoulPointItemStackModel SoulPointItemStack = new SoulPointItemStackModel();
+    public static final SubtitleModel Subtitle = new SubtitleModel();
```

Comment:
I think it might be better to make this model only care about spells. Subtitle does too many things to be a model.

Reply:
I did some changes, let me know if this is what you had in mind